tests/utils_tests/cleanup.py

```diff
@@ -0,0 +1,50 @@
+"""
+Test cleanup utilities for AQUA test suite.
+
+This module provides utilities for cleaning up files created during tests,
+preventing race conditions in parallel test execution.
+"""
+import os
+from aqua.core.logger import log_configure
+
+
+class TestCleanupRegistry:
+    """
+    Registry for tracking and cleaning up files created by tests.
+    
+    This class provides a simple way to clean up test artifacts
+    that are created in the AQUA configuration directory (~/.aqua).
+    """
+    
+    # Grid files created by tests that need cleanup
+    # Paths are relative to configdir (e.g., ~/.aqua/grids/regular.yaml)
+    FILES_TO_CLEAN = [
```

Comment:
perhaps you can move this files to the top of the files, so that it is simpler to see them if they need to me modified